src/components/aside/ProjectsMenuItem.vue

```diff
@@ -119,13 +148,36 @@ export default {
       }
     }
 
-    &__name {
+    &__info {
+      overflow-x: hidden;
+    }
+
+    &__name-wrapper {
+      display: flex;
```

Comment:
i think, it's not needed. Just put the badge inside the `__name` element and make it `inline-block`

Reply:
In name we use `v-html` prop. I don't know how to insert components inside v-html.

Reply:
> In name we use `v-html` prop. I don't know how to insert components inside v-html.

```vue
<div class="project-menu-item__name">
  <template v-html="name" /> 
  <project-badge />
</div>
```